### PR TITLE
add `docker compose config` command

### PR DESCRIPTION
**Pull request by asimpson**
`docker compose config` [docs](https://docs.docker.com/compose/reference/config/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Then select an action and follow the popup instructions.
 - docker network: rm
 - docker volume: rm
 - docker-machine: create, env, restart, rm, start, stop
-- docker-compose: build, create, down, exec, logs, pull, push, remove, restart, run, start, stop, up
+- docker-compose: build, create, down, exec, logs, pull, push, remove, restart, run, start, stop, up, config
```
Comment (Silex) [inline README.md]: Please sort alphabetically :wink:

File: docker-compose.el [hunk 2]
```diff
@@ -360,7 +366,9 @@
               (?S "Start"      ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-start-popup))
               (?T "Restart"    ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-restart-popup))
               (?U "Up"         ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-up-popup))
-              (?W "Down"       ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-down-popup))))
+              (?W "Down"       ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-down-popup))
+              (?V "Config"     ,(docker-utils-set-then-call 'docker-compose-arguments 'docker-compose-config))))
```
Comment (Silex) [inline docker-compose.el]: Again, please sort alphabetically

File: docker-compose.el [hunk 3]
```diff
@@ -189,6 +189,12 @@
   (interactive (list (docker-compose-read-services-names) (docker-compose-up-arguments)))
   (docker-compose--run-async "up" args services))
 
+;;;###autoload
+(defun docker-compose-config (args)
+  "Run \"docker-compose config ARGS."
```
Comment (Silex) [inline docker-compose.el]: It'd be ` "Run \"docker-compose config ARGS\"."` (misplaced dot)

**Comment by asimpson** [comment]
@Silex Fixed. Thanks for the feedback!

**Comment by Silex** [comment]
Thanks :+1:

**Comment by Silex** [comment]
Wait, does it actually do something? Maybe I merged too quickly here. I don't see a corresponding popup?

**Comment by asimpson** [comment]
@Silex yeah I couldn't really see what benefit the popup would add 🙈. I typically run `config` to see what the `.env` variables are evaluated as.

**Pull request merged**